tf2/package.xml

```diff
@@ -16,19 +17,17 @@
   <license>BSD</license>
 
   <url type="website">http://www.ros.org/wiki/tf2</url>
-    
-  <buildtool_depend version_gte="0.5.68">catkin</buildtool_depend>
 
-  <build_depend>libconsole-bridge-dev</build_depend>
-  <build_depend>geometry_msgs</build_depend>
-  <build_depend>rostime</build_depend>
-  <build_depend>tf2_msgs</build_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
 
-  <run_depend>libconsole-bridge-dev</run_depend>
-  <run_depend>geometry_msgs</run_depend>
-  <run_depend>rostime</run_depend>
-  <run_depend>tf2_msgs</run_depend>
+  <depend>console_bridge</depend>
+  <depend>geometry_msgs</depend>
+  
+  <test_depend>ament_cmake_gtest</test_depend>
+  <test_depend>ament_lint_auto</test_depend>
+  <!-- <test_depend>ament_lint_common</test_depend> -->
```

Comment:
Without any linters `ament_lint_auto` and the corresponding CMake code doesn't make much sense.